.storybook/helpers/files.data.ts

```diff
@@ -50,15 +70,26 @@ export const filesRoot: File[] = [
     ],
   },
   {
+    disabled: true,
     name: 'package.json',
   },
   {
     name: 'tsconfig.json',
   },
 ];
 
-export function getFileTreeNodeMarkup(files: File[] = filesRoot) {
+export function getFileTreeNodeMarkup(files: File[] = filesRoot, args = {}) {
   return files
-    .map(file => `<clr-tree-node>${file.name}${file.files ? getFileTreeNodeMarkup(file.files) : ''}</clr-tree-node>`)
+    .map(
+      file => `
+        <clr-tree-node
+        [clrDisabled]="${file.disabled}"
+        [clrExpanded]="${file.expanded}"
+        ${args['clrSelected'] !== undefined && file.selected !== undefined ? `[clrSelected]="${file.selected}"` : ''}
+        >
+            ${args['asLink'] ? `<a href="javascript:;" class="clr-treenode-link">${file.name}</a>` : file.name}
```

Comment:
```suggestion
            ${args.asLink ? `<a href="javascript:;" class="clr-treenode-link">${file.name}</a>` : file.name}
```

No need for string index access.

Reply:
corrected in commit e22dd2e